scheduler/src/main.rs

```diff
@@ -296,7 +296,9 @@ impl Manager {
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let config = ConfigParser::new()?;
-    env_logger::Builder::from_env(Env::default().default_filter_or(&config.verbosity_level)).init();
+    let subscriber = tracing_subscriber::fmt().compact().finish();
```

Comment:
if the logger is the same for `rikctl` / `controller` / `scheduler` / `riklet` why not move the logger initialisation inside a global common lib ?

Reply:
For now, it's to keep the governance of logger into the project, due to how critical it is.

Reply:
il will to it in https://github.com/rik-org/rik/issues/69